core_lib/src/tool/movetool.cpp

```diff
@@ -363,15 +367,11 @@ bool MoveTool::leavingThisTool()
     return true;
 }
 
-bool MoveTool::switchingLayer()
+void MoveTool::requestAction()
 {
-    auto selectMan = mEditor->select();
-    if (!selectMan->transformHasBeenModified())
-    {
-        mEditor->deselectAll();
-        return true;
-    }
+    if (mWarningShown) { return; }
```

Comment:
Isn’t this always reset to false at the end of this method anyway?

Reply:
Yes but UI dialogs block the execution loop so it's not reset until one has interacted with the dialog. The logic is from an older iteration of the PR though, it's redundant now and thus I've removed it.